packages/angular/cli/lib/cli/index.ts

```diff
@@ -12,8 +12,24 @@ import { runCommand } from '../../models/command-runner';
 import { getWorkspaceRaw } from '../../utilities/config';
 import { getWorkspaceDetails } from '../../utilities/project';
 
+export interface CliOptions {
+  testing?: boolean;
+  /**
+   * The cli arguments to provide to the command (without the `ng`)
+   */
+  cliArgs: string[];
```

Comment:
Can you change `cliArgs` to just `args`? Being a public api the name is now more important.